runtime/src/iree/builtins/ukernel/arch/arm_64/mmt4d_arm_64_dotprod.c

```diff
@@ -138,9 +140,9 @@ static inline void iree_uk_mmt4d_tile_s8s4s32_1x8x8_to_8x8x8_arm_64_dotprod(
         lhs[3] = vget_high_s8(lhs_uzp.val[1]);
       }
       // 4x8 * 8x8 -> 4x8.
-      for (int i = 0; i < 2; i++) {
-        for (int j = 0; j < 2; j++) {
-          for (int k = 0; k < 2; k++) {
+      IREE_UK_UNROLL for (int i = 0; i < 2; i++) {
+        IREE_UK_UNROLL for (int j = 0; j < 2; j++) {
+          IREE_UK_UNROLL for (int k = 0; k < 2; k++) {
```

Comment:
ouch, i just realized that this nested loop shadows the `k` from the enclosing outer loop. Let's avoid that!

Reply:
Replaced the k's with m's.